source/common/stats/symbol_table_impl.cc

```diff
@@ -442,19 +442,35 @@ void StatNameSet::rememberBuiltin(absl::string_view str) {
   builtin_stat_names_[str] = stat_name;
 }
 
-Stats::StatName StatNameSet::getStatName(absl::string_view token) {
+StatName StatNameSet::getBuiltin(absl::string_view token, StatName fallback) {
+  StatName stat_name = getBuiltinHelper(token, fallback);
+  if (stat_name == fallback) {
```

Comment:
perf nit: you could likely avoid the comparison here by potentially returning a struct from the helper (or something like that) as to whether it selected the fallback?